includes/wc-update-functions.php

```diff
@@ -2091,6 +2091,13 @@ function wc_update_400_increase_size_of_column() {
 	$wpdb->query( "ALTER TABLE {$wpdb->prefix}wc_product_meta_lookup MODIFY COLUMN `max_price` decimal(19,4) NULL default NULL" );
 }
 
+/**
+ * Reset ActionScheduler migration status. Needs AS >= 3.0 shipped with WC >= 4.0.
+ */
+function wc_reset_action_scheduler_migration_status() {
+	\ActionScheduler_DataController::mark_migration_incomplete();
```

Comment:
This should have a check that the method exists. It's possible that the "active" version of AS is not the version included in WC.

This looks good to me but I'll leave PR approval for the core team.

Reply:
Added checks around. Thanks for review @rrennick !